components/style/properties/properties.mako.rs

```diff
@@ -313,6 +313,42 @@ impl PropertyDeclarationBlock {
     }
 }
 
+impl PropertyDeclarationBlock {
+    // Take a declaration block known to contain a single property,
+    // and serialize it
+    pub fn to_css_single_value<W>(&self, dest: &mut W, name: &str)
+        -> fmt::Result where W: fmt::Write {
+        match self.declarations.len() {
+            0 => Err(fmt::Error),
+            1 if self.declarations[0].0.name().eq_str_ignore_ascii_case(name) => {
```

Comment:
I was about to r+ this, but don't you want something like:

```rust
1 => {
    if !self.declarations[0].0.name().eq_str_ignore_ascii_case(name) {
        return Err(fmt::Error);
    }
    self.declarations[0].0.to_css(dest)
}
```

I think it's not going to matter because it will get rejected in the `all` condition below, but maybe it's cleaner? In any case, r=me